### PR TITLE
CBG - Frontend / behavior for print checklist tool

### PR DESCRIPTION
**Pull request by el-mapache**
So that the user can generate a list of items to tackle, as well as a list of other things to keep in mind when thinking about buying a car, create an interface to allow them to select items, and print the lists.

## Additions

- Adds JS and CSS to handle the car buying guide checklist functionality

## Testing

0. In the admin, navigate to the `buying a car` page. At the bottom, add a YESChecklist, and populate a few boxes. Publish!

1. Select some checkboxes at the bottom of the page:
  * You should only be able to select 3 options at a time
  * Selecting a fourth deselects the last checked element
  * Selecting a checked element deselects it
2. Push the `print` button:
  * The dialog should open a print view with two tables. The top table is populated with the checkboxes the user selected, while the bottom table is populated with the remaining checkboxes

## Screenshots

Screen Shot 2019-10-31 at 9 14 15 AM

Screen Shot 2019-10-31 at 9 15 06 AM

## Checklist

- [ ] PR has an informative and human-readable title
- [ ] Changes are limited to a single goal (no scope creep)
- [ ] Code can be automatically merged (no conflicts)
- [ ] Code follows the standards laid out in the [CFPB development guidelines](https://github.com/cfpb/development)
- [ ] Passes all existing automated tests
- [ ] Any _change_ in functionality is tested
- [ ] New functions are documented (with a description, list of inputs, and expected output)
- [ ] Placeholder code is flagged / future todos are captured in comments
- [ ] Visually tested in supported browsers and devices (see checklist below :point_down:)
- [ ] Project documentation has been updated
- [ ] Reviewers requested with the [Reviewers tool](https://help.github.com/articles/requesting-a-pull-request-review/) :arrow_right:

### Browsers

- [ ] Chrome on desktop
- [ ] Firefox
- [ ] Safari on macOS
- [ ] Edge
- [ ] Internet Explorer 9, 10, and 11
- [ ] Safari on iOS
- [ ] Chrome on Android

### Accessibility

- [ ] Keyboard friendly
- [ ] Screen reader friendly

### Other

- [ ] Is useable without CSS
- [ ] Is useable without JS
- [ ] Flexible from small to large screens
- [ ] No linting errors or warnings
- [ ] JavaScript tests are passing

### CODE REVIEW

**Comment by el-mapache** [comment]
@anselmbradford I'd love a look at this tomorrow morning if you have time, I'm also happy to demo it using my local env if that is easier than adding content into wagtail.

File: cfgov/unprocessed/css/pages/car-buying-guide.less [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+  thead {
+    th, td {
+      margin-bottom: unit( 14px / @base-font-size-px, em );
+      font-weight: 600;
+      letter-spacing: 1px;
+      line-height: 1.25;
+      text-transform: uppercase;
+      padding: unit( 12px / @base-font-size-px, em );
+      color: #101820;
```
[the remark this answers is not in the document]
Comment (el-mapache) [reply cfgov/unprocessed/css/pages/car-buying-guide.less]: Ah good call, thanks!

File: cfgov/unprocessed/apps/youth-employment-success/js/views/print-button.js [hunk 2]
```diff
@@ -45,13 +54,24 @@ function printButton( element ) {
       .forEach( el => el.classList.add( `${ CLASSES.HIDE }` ) );
 
     window.addEventListener( 'focus', _onAfterPrint );
+
     window.print();
   }
 
+  function _handleClick( event ) {
+    console.log('handlee')
```
[the remark this answers is not in the document]
Comment (el-mapache) [reply cfgov/unprocessed/apps/youth-employment-success/js/views/print-button.js]: Yes definitely! 😬

**Comment by el-mapache** [comment]
@anselmbradford @Scotchester Removed JSON block, now just querying the DOM for the data I need!

**Comment by el-mapache** [comment]
hey @anselmbradford just bumping ya on this 😄 Just want to make sure we have a little buffer to address issues on friday, if necessary

**Comment by anselmbradford** [comment]
An issue was flagged by @chosak with this that the YESChecklist component can be added to any browse page, but unless it's at the specific URL specified in the route it will appear on the page without its JavaScript. We probably don't want that ability and instead want to make the JS "component-level" (description [here](https://github.com/cfpb/cfgov-refresh/pull/5291#issuecomment-549925681)), right @chosak? Sorry I led you astray on this one 😞

**Comment by el-mapache** [comment]
@anselmbradford I was in a meeting with @Scotchester and @chosak and it seemed like we agreed to remove the model code. That code isn't present here, but was in a branch I was walking through with them.

It would be really helpful to just get a straight forward breakdown of what I need to do to get this PR in a good state, probably from @chosak?

**Pull request merged**